agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/Constants.kt

```diff
@@ -2,6 +2,7 @@
 package com.sourcegraph.cody.agent.protocol_generated;
 
 object Constants {
+  const val `` = ".*"
```

Comment:
```suggestion
  const val  MATCH_ALL = ".*"
```

See[ the fix](https://github.com/sourcegraph/cody/pull/6598/files#r1916204716)